epub33/a11y-tech/index.html

```diff
@@ -1680,6 +1690,338 @@ <h4>Identify the pagination source</h4>
+             src="chapter1.xhtml#fn01"/>
+         &lt;audio
+             src="chapter1_audio.mp3"
+             clipBegin="0:24:15.000"
+             clipEnd="0:24:53.821"/>
+      &lt;/par>
+
+      &lt;!-- another paragraph -->
+      &lt;par id="id3">
+         &lt;text
+             src="chapter1.xhtml#para2"/>
+         &lt;audio
+             src="chapter1_audio.mp3"
+             clipBegin="0:24:53.821"
+             clipEnd="0:25:28.530"/>
+      &lt;/par>
+   &lt;/body>
+&lt;/smil>
+</pre>
+
+						<p>EPUB Content Document:</p>
+
+						<pre>&lt;html … >
+   …
+   &lt;body>
+      &lt;p id="para1">
+         This is the paragraph with the footnote reference …
+      &lt;/p>
+      
+      &lt;p id="fn01" role="doc-footnote">
+         This is the footnote …
+      &lt;/p>
+      
+      &lt;p id="para2">
+         This is the paragraph after the footnote …
+      &lt;/p>
+   &lt;/body>
+&lt;/html></pre>
+					</aside>
+				</section>
+
+				<section id="sync-004">
+					<h4>Identifying escapable structures</h4>
+
+					<p>Some content elements are containers for expressing complex information. A table, for example,
+						has data arranged in rows and cells, where each cell may contain several paragraphs of
+						information. Lists similarly may contain many items, and each item can contain complex
+						information. While users may be interested in some of the information in these structures, they
+						also often want to escape from them to keep reading, not have to listen to the entire content
+						before being able to move on. These are called escapable elements, because the user needs to be
+						able to escape from them whenever they choose to simplify the reading experience.</p>
+
+					<p>EPUB 3's <a data-cite="epub-33#sec-media-overlays">Media Overlays feature</a> [[EPUB-33]] does
+						not allow Reading Systems to determine if playback sequences are escapable unless EPUB Creators
+						add additional semantics to the markup, however. EPUB Creators must use the <a
+							data-cite="epub-33#sec-epub-type-attribute"><code>epub:type</code> attribute</a> [[EPUB-33]]
+						to add semantics to <a data-cite="epub-33#sec-smil-seq-elem"><code>seq</code></a> and <a
+							data-cite="epub-33#sec-smil-seq-elem"><code>par</code></a> elements [[EPUB-33]] to allow
+						Reading Systems to provide users the option to escape their playback sequences.</p>
+
+					<p>The recommended structures to identify for escapability are:</p>
+
+					<ul>
+						<li>Figures &#8212; use the <a data-cite="epub-ssv-11#figure"><code>figure</code></a> semantic
+							[[EPUB-SSV-11]] to identify each figure.</li>
+						<li>Lists &#8212; use the <a data-cite="epub-ssv-11#list"><code>list</code></a> and <a
+								data-cite="epub-ssv-11#list-item"><code>list-item</code></a> semantics [[EPUB-SSV-11]]
+							to allow escaping from both items and lists.</li>
+						<li>Tables &#8212; use the <a data-cite="epub-ssv-11#table"><code>table</code></a>, <a
```

Comment:
As in the A11y doc, I am a little uneasy about table escapability and in general, nested escapability. I don't want to say that it's not possible, I just don't think nested escapability is implemented anywhere (though I haven't looked into it, it's just a hunch). We don't provide good implementor guidance either, just a basic "this is how you escape something" rule in the [reading systems spec](https://w3c.github.io/epub-specs/epub33/rs/#sec-escapability). Content creators may still want to aim high with their markup, so that's a reason to keep this section as-is; but realistically IMO not worth the trouble to markup MO that extensively.

Reply:
Oh, I guess this partly answers my question above. It would still help to see if we can get consensus on what to do about table markup. I've removed tables from the understanding description for now.